### ISSUE TITLE
onCellEditingStopped triggered twice after set stopEditingWhenGridLosesFocus to true

### ISSUE THREAD

**Issue by Quc890925** [ag-grid/ag-grid#2402]
**I'm submitting a defetct regarding ag-grid stopEditingWhenGridLosesFocus and onCellEditingStopped callback **  

**Current behavior**
            this.gridOptions.stopEditingWhenGridLosesFocus = true;
            this.gridOptions.onCellEditingStopped = (event) => this.checkInputValue(event);
I'm using Angular 6 and ag-grid and tried to have a callback function after a cell is edited, but after I added stopEditingWhenGridLosesFocus value as "true" the onCellEditingStopped call back(checkInputValue function) will be triggered twice.

**Expected behavior**
 the onCellEditingStopped call back(checkInputValue function) should be triggered only once as it'll be stopped only once.

**Minimal reproduction of the problem with instructions**
            this.gridOptions.stopEditingWhenGridLosesFocus = true;
            this.gridOptions.onCellEditingStopped = (event) => this.checkInputValue(event);

**What is the motivation / use case for changing the behavior?**
call a function when grid cell loses focus and edit status is stopped

**Please tell us about your environment:**
VS code, Angular 6

* **ag-Grid version:** 17.1.1

* **Browser:**
IE, Chrome

* **Language: 
TypeScript

**Comment by makinggoodsoftware** [comment]
Hi,

Can you provide a reproducible scenario using one of our examples or your source code in a plunker and we will investigate?

Accepted reproducible scenarios are
- A description of the detailed steps to reproduce your behaviour in one of our examples in the docs.
- A plunker

If you decide to send us a plunkr, from any example in our website use the plunkr button in there to fork your own code by following the steps below:

- Select the framework that is appropriate to you from the drop-down
- Open it in plunker. (Use the button plunker in our example)
- Add your changes so that the behaviour is reproduced
- Save and Freeze the plunker(On the top left corner)
- Send us the link to the plunker(You can copy the URL from the browser)

If reporting a bug make sure to state.

Actual behaviour.
Expected behaviour. If possible back this up with our docs/examples if possible
***

Note that this issue is getting closed, but you are welcomed to create a new issue if you can provide us with a reproducible scenario

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.
